modules/outbound-command-event-handler/src/domain/individual_transfer_entity.ts

```diff
@@ -0,0 +1,108 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list (alphabetical ordering) of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+ * Modusbox
+ - Vijay Kumar Guthi <vijaya.guthi@modusbox.com>
+ --------------
+ ******/
+
+'use strict';
+
+import { BaseEntityState, BaseEntity, AjvValidationError } from '@mojaloop/sdk-scheme-adapter-public-shared-lib';
+import { SDKSchemeAdapter, v1_1 as FSPIOP } from '@mojaloop/api-snippets';
+import { randomUUID } from 'crypto';
+import Ajv from 'ajv';
+const ajv = new Ajv({
+    strict:false,
+    allErrors: false,
+});
+
+// TODO: check name and status enums.
+export enum IndividualTransferInternalState {
+    RECEIVED = 'RECEIVED',
+    DISCOVERY_PROCESSING = 'DISCOVERY_PROCESSING',
+    AGREEMENT_PROCESSING = 'AGREEMENT_PROCESSING',
+    TRANSFER_PROCESSING = 'TRANSFER_PROCESSING',
+}
+
+// TODO: Standardize the following
+export interface IHttpRequest {
+    method: string;
```

Comment:
What about params?

It is very likely these will be parsed for us already, so we might as well include it.

Reply:
Removed this unnecessary interface